### DISCUSSION TITLE
Why my github copilot is asking for bill, when my student developer pack is still active?

### DISCUSSION THREAD

**Discussion by MarufHasan24** [community/community#43672] [GitHub Education]
Suddenly, today morning I saw that my Github Copilot is asking for paying bill. But it's working nicely until last night. Why it's asking for bill, student developer pack is still active?

**Comment by danbishop-nz** [comment]
I have also had the same issue

**Comment by mijanConnect** [comment]
I'm also facing the same problem.

**Comment by sunjiabin17** [comment]
same too

**Comment by otkrickey** [comment]
Solved: #43634

**Comment by Hleonor** [comment]
the same too

**Comment by Feliks9527** [comment]
too

**Comment by sunjiabin17** [comment]
<img width="555" alt="image" src="https://user-images.githubusercontent.com/41134681/211264375-2663fbb6-7bbe-457f-b00c-ee6b48e17a77.png">

**Comment by Feliks9527** [comment]
yo, um.So I got your message I wish you would answer the fuckin' phone.Um...No, I don't plan to reply to everybody.um...I'll hit you back later......

**Comment by jach8** [comment]
A simple refresh of the vscode app made it work for me. Disable, and Enable the extension.

**Comment by esaldor** [comment]
I have also had the same issue

**Comment by Actticus** [comment]
same

**Comment by ross39** [comment]
same

**Comment by hon2838** [comment]
same

**Comment by BANZOM** [comment]
It arises today in my account.

**Comment by Feliks9527** [comment]
yo, um.So I got your message I wish you would answer the fuckin' phone.Um...No, I don't plan to reply to everybody.um...I'll hit you back later......

**Comment by samus-aran** [comment]
I'll be closing this discussion due to lack of recent activity. If you still haven't received your Global Campus benefits please feel free to re-open.
